lib/routes/routeBackbeat.js

```diff
@@ -1388,7 +1388,7 @@ function routeBackbeat(clientIP, request, response, log) {
         const path = request.url.replace('/_/backbeat/api', '/_/');
         const { host, port } = config.backbeat;
         const target = `http://${host}:${port}${path}`;
-        return auth.server.doAuth(request, log, (err, userInfo) => {
+        return auth.server.doAuth(request, log, (err, userInfo, streamingV4Params, infos) => {
```

Comment:
not the right one to change: this is used when forwarding to backbeat api pod...

the auth call for "cloudserver" backbeat routes is below...